packages/mask/src/components/InjectedComponents/ProfileTabContent.tsx

```diff
@@ -93,32 +204,33 @@ export function ProfileTabContent(props: ProfileTabContentProps) {
         label: typeof x.label === 'string' ? x.label : translate(x.pluginID, x.label),
     }))
 
-    const selectedTabId = selectedTab ?? first(tabs)?.id
+    const [currentTab, onChange] = useTabs(first(tabs)?.id ?? PluginId.Collectible, ...tabs.map((tab) => tab.id))
 
-    const { value: currentVisitingSocialIdentity, loading: loadingCurrentVisitingSocialIdentity } =
-        useCurrentVisitingSocialIdentity()
+    const isWeb3ProfileDisable = useIsMinimalMode(PluginId.Web3Profile)
 
     const showNextID = !!(
         isTwitter(activatedSocialNetworkUI) &&
-        isOwnerIdentity &&
-        (socialAddressList.length === 0 || !currentVisitingSocialIdentity?.hasBinding)
-    )
-    const handleOpenDialog = () => {
-        CrossIsolationMessages.events.requestWeb3ProfileDialog.sendToAll({
-            open: true,
-        })
-    }
-    const Component = getTabContent(
-        showNextID ? displayPlugins?.find((tab) => tab?.pluginID === PluginId.NextID)?.ID : selectedTabId,
+        (isWeb3ProfileDisable ||
+            (isOwnerIdentity && !currentVisitingSocialIdentity?.hasBinding) ||
+            (isOwnerIdentity &&
+                socialAddressList.findIndex((address) => address.type === SocialAddressType.NEXT_ID)) === -1 ||
+            !socialAddressList?.length)
```

Comment:
This condition is redundant.

```suggestion
                socialAddressList.findIndex((address) => address.type === SocialAddressType.NEXT_ID)) === -1)
```